### DISCUSSION TITLE
New tags convention causes problem for plugin pl.zalas.structurizr-cli

### DISCUSSION THREAD

**Discussion by albert-dabrowski** [structurizr/cli#160] [General]
I saw that starting this year there is a new convention for tagging. Previously it was vnn.nn.nn. Now it's a date and it missing 'v' at the beginning which was assumed in the gradle plugin thus unable to download it.
Is that new tagging convention being introduced by purpose. I suppose if you add tag with 'v' at the beginning it would work still

**Comment by simonbrowndotje** [comment]
I think the `v` was missing on that first release because of a typo in the build script ... it's back again now though.
